### PR TITLE
this commit will return the topics

### PR DESCRIPTION
**Pull request by AdamG63**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+
+const request = require('supertest');
+const { response } = require('../app');
```
Comment (Rosem10) [inline __tests__/app.test.js:2]: We don't need this line!

File: model/topics.model.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+const db = require('../db/connection');
+
+exports.fetchTopics = () => {
+    return db.query('SELECT * FROM topics').then((result) => {
+       console.log(result.rows, 'in the model')
```
Comment (Rosem10) [inline model/topics.model.js]: Remove console.logs please!

File: app.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+const express = require('express');
+const { getTopics } = require('./controller/topics.contoller');
+const app = express();
+
+app.get('/api/topics', getTopics);
+
```
Comment (Rosem10) [inline app.js:6]: Please add an initial bit of error handling for the errors that aren't going to be attached to specific endpoints (e.g 500 internal server error and 404 endpoint not found).

**Review by Rosem10 — CHANGES_REQUESTED** [review]

File: app.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+app.all('/*', (req, res) => {
+    res.status(404).send({msg: 'Not found'})
+})
+app.use((err, req, res, next) => {
+    res.status(500).send('internal server errow');
```
Comment (mallionaire) [inline app.js]: Whoops, typo... error, not errow

File: __tests__/app.test.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+        body.topics.forEach((topic) => {
+            expect.objectContaining({
+                description: expect.any(String),
+                slug: expect.any(String)
+            })
+        })
+    })
+
+})
+test('404: responds with not found when passed with the wrong endpoint', () => {
```
Comment (mallionaire) [inline __tests__/app.test.js]: If I was being really picky I'd say "wrong endpoint" is a misleading description here, invalid endpoint is more appropriate to use.

**Review by mallionaire — APPROVED** [review]
Looking good, I've pointed out a couple of teeny changes, like a typo. 
Please correct those and then you're ok to commit and merge in (I won't insist on another pull req as they're nothing major!)

File: __tests__/app.test.js (hunk 5 above)
Comment (AdamG63) [reply __tests__/app.test.js]: thank you 👍

**Pull request merged**